test/e2e/framework.go

```diff
@@ -877,7 +892,21 @@ func (data *TestData) deployFlowAggregator(ipfixCollector string) error {
 	if err != nil || rc != 0 {
 		return fmt.Errorf("error when deploying the Flow Aggregator; %s not available on the control-plane Node", flowAggYaml)
 	}
-	if err = data.mutateFlowAggregatorConfigMap(ipfixCollector); err != nil {
+	if o.secureConnection {
```

Comment:
Maybe add some comments  explaining we need to copy the secret to flow-aggregator namespace to make it accessible by FA?

Reply:
I added a comment there. Please take a look to see if it makes sense? Thanks.